zerver/models.py

```diff
@@ -2332,6 +2332,9 @@ class UserActivity(models.Model):
     class Meta:
         unique_together = ("user_profile", "client", "query")
 
+def get_latest_update_message_flag_activity(user_profile: UserProfile) -> Optional[UserActivity]:
+    return UserActivity.objects.filter(user_profile=user_profile, query='update_message_flags').last()
```

Comment:
Let's add a comment at the top of `update_user_activity` that if we rename that function, we need to change this, just as a reminder.